shopfloor/services/zone_picking.py

```diff
@@ -160,9 +160,21 @@ def _response_for_start(self, message=None):
         zones = self.work.menu.picking_type_ids.mapped(
             "default_location_src_id.child_ids"
         )
+        data = {"zones": self._data_for_select_zone(zones)}
+        buffer = self._find_buffer_move_lines()
+        if buffer:
+            # Some lines can be unloaded, let the user know
+            # The call to the endpoint will need the location and picking id
+            line = first(buffer)
+            picking_type = line.picking_id.picking_type_id
+            zone = line.move_id.location_id
+            data["buffer_is_loaded"] = {
```

Comment:
having a buffer already means we loaded something
```suggestion
            data["buffer"] = {
```
could be enough?

Reply:
could be :shrug:

Reply:
I agree, can you rename it to "buffer"?

Reply:
Changed.